lib/income.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Plaid.Income do
+  @moduledoc """
+  Functions for Plaid `income` endpoint.
+  """
+
+  import Plaid, only: [make_request_with_cred: 4, get_cred: 0]
+
+  alias Plaid.Utils
+
+  @derive Jason.Encoder
+  defstruct item: nil, income: nil, request_id: nil
+
+  @type t :: %__MODULE__{
+          item: Plaid.Item.t(),
+          income: Plaid.Income.Income.t(),
+          request_id: String.t()
+        }
+  @type params :: %{
+          required(:access_token) => String.t()
+        }
+  @type cred :: %{required(atom) => String.t()}
+
+  @endpoint "income"
+
+  defmodule Income do
+    @moduledoc """
+    Plaid Income Income data structure.
```

Comment:
Could you change this to `Plaid.Income Income data structure`? I'm not happy with how my naming convention is shaking out, but this will help add clarity and look less like a typo.

Reply:
I think I interpreted this comment correctly to update the `@moduledoc` here. I also honored the request here:
https://github.com/wfgilman/plaid-elixir/pull/16/files#diff-4232bda31bc3e3b13d7d9a1716fecbb2R51

I moved the other files per your request as well. Glad you caught that, I meant to put those files in their respective `plaid` folders 😊

Let me know if there's anything more.

Thanks so much!

John